### DISCUSSION TITLE
taxa showing up as significant in global results, but are not different in primary or pairwise results ancombc2

### DISCUSSION THREAD

**Discussion by anya-mueller** [FrederickHuangLin/ANCOMBC#219] [Q&A]
Hello,

I am conducting a differential abundance analysis with my microbiome data using anocombc2 and am encountering what appears to be a strange result. I was looking at the taxa identified as significantly differentially abundant with the global analysis in my primary and pairwise results and I encountered instances where there was no significant differences detected in the primary and pairwise results (ie contains("diff") = FALSE for all). Why might this be the case? Should I be concerned?

I was trying to look at the difference in how these three analyses are conducted but was running into the wall of internal functions. Would you be able to explain the differences to me?

Thank you for your time and help.

**Comment by FrederickHuangLin** [comment]
Hello @anya-mueller,

This is an excellent question!

The discrepancy you've observed between the global test and pairwise test results is likely attributable to a power issue. The global test identifies taxa that are differentially abundant in at least one group across all the different groups but does not specify which particular groups are different. This characteristic makes the global test more statistically powerful than the pairwise test. It's important to note that this difference is not due to the code but rather stems from the fundamental dissimilarity between these two approaches.

If you are indeed facing power limitations, one approach is to report the significant taxa identified by the global test and then determine the direction of their abundance using the effect sizes reported by the pairwise test. Be sure to acknowledge this limitation in your statistical analysis and transparently communicate it to reviewers and readers.

Best regards,
Frederick

**Reply by anya-mueller** [reply]
Thanks Frederick!
